src/site/content/en/fast/font-best-practices/index.md

```diff
@@ -0,0 +1,480 @@
+of effort required to apply these optimizations will depend somewhat on the
+languages that your site supports. In particular, be aware that optimizing fonts
+for [CJK languages](https://en.wikipedia.org/wiki/CJK_characters) can be
+particularly challenging.
+
+
+*  **Unicode-range and font subsetting:** `@font-face` is often used in
+   conjunction with the
+   [`unicode-range`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range)
+   descriptor. `unicode-range` informs the browser which characters a font can
+   be used for.
+
+
+    ```css
+    @font-face {
+    font-family: "Open Sans";
+    src: url("/fonts/OpenSans-Regular-webfont.woff2") format("woff2");
+    unicode-range: U+0025-00FF;
+    }
+    ```
+
+    A font file will be downloaded if the page contains one or more characters
+    matching the unicode range. `unicode-range` is commonly used to serve
+    different font files depending on the language used by page content.
+
+    `unicode-range` is often used in conjunction with the technique of
+    subsetting. A subset font includes a smaller portion of the
+    [glyphs](https://en.wikipedia.org/wiki/Glyph) (that is, characters) that
+    were contained in the original font file. For example, rather than serve all
+    characters to all users, a site might generate separate subset fonts for
+    Latin and Cyrillic characters. The number of glyphs per font varies wildly:
+    Latin fonts are usually on the magnitude of 100 to 1000 glyphs per font;
+    [CJK](https://en.wikipedia.org/wiki/CJK_characters) fonts may have over
+    10,000 characters. Removing unused glyphs can significantly reduce the
+    filesize of a font.
+
+    Tools for generating font subsets include
+    [subfont](https://github.com/Munter/subfont) and
+    [glyphanger](https://github.com/zachleat/glyphhanger).
+
+    For information on how Google Fonts implements font subsetting, see this
+    [presentation](https://www.unicodeconference.org/presentations-42/S5T3-Sheeter.pdf).
+    For the Google Fonts API subsets, see this
+    [repo](https://github.com/googlefonts/gftools/tree/main/Lib/gftools/encodings).
+
+*  **WOFF2:** Of the modern font fonts, [WOFF2](https://www.w3.org/TR/WOFF2/) is
+   the newest, has the widest browser support, and offers the best compression.
+   Because it uses Brotli, WOFF2 compresses 30% better than WOFF.
+
+
+
+#### Use fewer web fonts
+
+The fastest font to deliver is a font that isn't requested in the first place.
+System fonts and variable fonts are two ways to potentially reduce the number of
+web fonts used on your site.
+
+
+*  **System fonts:** A system font is the default font used by the user
+   interface of a user's device. System fonts typically vary by operating system
+   and version. Because the font is already installed, the font does not need to
+   be downloaded. System fonts can work particularly well for body text. 
```

Comment:
```suggestion
A **system font** is the default font used by the user
interface of a user's device. System fonts typically vary by operating system
and version. Because the font is already installed, the font does not need to
be downloaded. System fonts can work particularly well for body text. 
```